services/fncsgossbridge/service/fncs_goss_bridge.py

```diff
@@ -1411,10 +1410,10 @@ def _keep_alive(is_realtime, archive_db_file, archive_file, only_archive):
 
 
 def _main(simulation_id, simulation_broker_location='tcp://localhost:5570', measurement_map_dir='', is_realtime=True,
-          sim_duration=86400, sim_start=0, archive_db_file=None, archive_file=None, only_archive=False):
+          sim_duration=86400, sim_start=0, archive_db_file=None, archive_file=None, only_archive=False, username='system', password='manger'):
```

Comment:
doesn't this hard code the username and password?  Shouldn't we get this from an environmental variable somewhere?